pyaedt/application/Variables.py

```diff
@@ -1936,10 +1949,11 @@ def __add__(self, other):
         >>> assert result.unit_system == "Current"
 
         """
-        assert isinstance(other, Variable), "You can only add a variable with another variable."
-        assert (
-            self.unit_system == other.unit_system
-        ), "Only ``Variable`` objects with the same unit system can be added."
+        if not isinstance(other, Variable):
+            raise ValueError("You can only add a variable with another variable.")
+        if self.unit_system != other.unit_system:
+            raise ValueError("Only ``Variable`` objects with the same unit system can be added.")
```

Comment:
I thought you couldn't use double backticks inside a message string? Other messages use a single quotation mark (``'``). I see double backticks here and in the next error message.

Reply:
Maybe it has side effects once rendered, thanks for the warning.
The python interpretation is working though
![image](https://github.com/ansys/pyaedt/assets/146729917/f050a79e-d23c-40e0-9038-f0d3feaa2f87)